internal/ascprails/postgres_integration_test.go

```
@@ -0,0 +1,247 @@
+		t.Fatalf("first=%d replays=%d", first, replays)
+	}
+	gate, err := NewPostgresOperationGate(db)
+	if err != nil || gate.Check(context.Background(), enqueued) != nil {
+		t.Fatalf("operation gate: %v", err)
+	}
+	changedGate := enqueued
+	changedGate.Binding.CommitmentHash = testHash("99")
+	if err := gate.Check(context.Background(), changedGate); !errors.Is(err, ErrOperationNotExecutable) {
+		t.Fatalf("substituted operation gate=%v", err)
+	}
+	mutation, err := db.Begin()
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).Begin must not be called. use (*database/sql.DB).BeginTx

(noctx)